src/argilla/tasks/workspaces/create.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import typer
+
+from argilla.client.workspaces import Workspace
+from argilla.tasks import async_typer
+
+
+def create_workspace(
+    workspace_name: str,
+):
```

Comment:
I think you can just rename the arg to `name` as `workspace_name` may be redundant if the function is already named `create_workspace`

```suggestion
def create_workspace(name: str = typer.Option(..., prompt=True, help="The name of the workspace to be created")) -> None:
```

Reply:
Adding typer Options would require the user to run the command as workspaces create --name <workspace name> right?

Reply:
Right, and with the `prompt=True`, if not provided then it will be asked to the user via the prompt 👍🏻

Reply:
For reference I'd suggest you to look into the `users create` command recently developed by @gabrielmbmb and merged into `develop`

Reply:
I would change `typer.Option` to `typer.Argument`